diff --git a/runtime/src/atn/ATNState.h b/runtime/src/atn/ATNState.h
index 8ed613735..b6f2eefd2 100755
--- a/runtime/src/atn/ATNState.h
+++ b/runtime/src/atn/ATNState.h
@@ -70,7 +70,7 @@ namespace atn {
   ///
   /// <embed src="images/OptionalNonGreedy.svg" type="image/svg+xml"/>
   /// </summary>
-  class ANTLR4CPP_PUBLIC ATN;
+  // class ANTLR4CPP_PUBLIC ATN;
 
   class ANTLR4CPP_PUBLIC ATNState {
   public:
diff --git a/runtime/src/support/Any.h b/runtime/src/support/Any.h
index 468db9894..46d0ea842 100644
--- a/runtime/src/support/Any.h
+++ b/runtime/src/support/Any.h
@@ -117,7 +117,7 @@ private:
 
     T value;
 
-    Base* clone() const {
+    Base* clone() const override {
       return clone<>();
     }
 
